blockchain.go

```diff
@@ -24,19 +24,22 @@ type Signidice struct {
 	Signature string `json:"sign"`
 }
 
-
-func GetSigndiceTransaction(api *eos.API, contract, casinoAccount string,requestID uint64, signature string) (*eos.SignedTransaction, *eos.PackedTransaction, error) {
+func GetSigndiceTransaction(
+	api *eos.API,
+	contract, casinoAccount string,
+	requestID uint64, signature string,
+) (*eos.SignedTransaction, *eos.PackedTransaction, error) {
 	action := NewSigndice(contract, casinoAccount, requestID, signature)
 	txOpts := &eos.TxOptions{}
 
 	if err := txOpts.FillFromChain(api); err != nil {
 		return nil, nil, err
 	}
 	tx := eos.NewTransaction([]*eos.Action{action}, txOpts)
+	// eos-go will automatically sign with required key
 	signedTx, packedTx, err := api.SignTransaction(tx, txOpts.ChainID, eos.CompressionNone)
```

Comment:
Here is better to use `api.Signer.Sign` directly. The `api.SignTransaction` make unnecessary call to node api: https://github.com/eoscanada/eos-go/blob/master/api.go#L362 .
Each call to node API take approximately 200-400ms, so for us it really important to be faster